.github/workflows/gating.yaml

```diff
@@ -8,6 +8,15 @@ on:
   workflow_dispatch:
     inputs: {}
 
+env:
+  IMAGE_NAME: cachi2
+  IMAGE_REGISTRY: quay.io
+  IMAGE_NAMESPACE: containerbuildsystem
+  IMAGE_TAG: latest
+  TEST_REPO: https://github.com/release-engineering/retrodep.git
+  TEST_BRANCH: v2.1.1
+  TEST_FOLDER: /tmp/source
```

Comment:
Might just be me, but I prefer fewer variables with a clearer purpose

The TEST variables don't feel super necessary, since they are used only within the single smoketest step. When we add integration tests, they might even become confusing

Not sure about the IMAGE variables, they make more sense than the test ones but I kinda prefer having the whole pullspec in one place to make it `grep`-able

Reply:
+1 for making pullspec `grep`-able .. I am ok with removing vars completely if you don't mind changing these vars on many places during refactoring

Reply:
I personally value readability over refactor-ability in CI definitions. Otherwise I end up having to mentally substitute the variables when reading the tasks, and refactoring is usually a search+replace operation for me anyway :smile: 

But I'd like others' opinions on this as well

Reply:
if no one will object I'll remove variables :)

Reply:
+1

Reply:
vars removed